website/docs/docs/dbt-cloud-apis/sl-graphql.md

```diff
@@ -0,0 +1,106 @@
+---
+title: "GraphQL"
+id: sl-graphql
+description: "Integrate and use the GraphQL API to query your metrics."
+tags: [Semantic Layer, APIs]
+---
+
+<VersionBlock lastVersion="1.5">
+
+import LegacyInfo from '/snippets/_legacy-sl-callout.md';
+
+<LegacyInfo />
+
+</VersionBlock>
+
+
+[GraphQL](https://graphql.org/) (GQL) is an open-source query language for APIs. It offers a more efficient and flexible approach compared to traditional RESTful APIs. 
+
+With GraphQL, users can request specific data using a single query, reducing the need for many server round trips. This improves performance and minimizes network overhead.
+
+GraphQL has several advantages, such as self-documenting, having a strong typing system, supporting versioning and evolution, enabling rapid development, and having a robust ecosystem. These features make GraphQL a powerful choice for APIs that prioritize flexibility, performance, and developer productivity.
+
+dbt Partners can use the Semantic Layer GraphQL API to build and integration with the dbt Semantic Layer.
+
+## dbt Semantic Layer GraphQL API
+
+The dbt Semantic Layer GraphQL API allows you to explore and query metrics and dimensions. Due to it's self-documenting nature, you can explore the calls conveniently through the [schema explorer](https://cloud.getdbt.com/semantic-layer/api/graphql). 
+
+## Using the GraphQL API
+
+If you are a dbt user or partner with access to dbt Cloud and the Semantic Layer, you can setup and test this API with data from your own instance by configuring the Semantic Layer and obtaining the right GQL connection parameters described in this document. (PROVIDE link to set up?). 
```

Comment:
@mirnawong1  here I was thinking that we link back to set up of the Semantic Layer or even the quickstart?. I'd imagine mostly partners will use this, but they may want to know that there's a way they can set this up in their own cloud instance.